tools/jsonc_gen_lport_groups.sh

```diff
@@ -221,6 +220,7 @@ function emit_jsonc_lport_groups
             "threads": ["fwd:$i"],
             "pmd": "net_af_xdp",
             "skb_mode": $CNDP_COPY_MODE,
+            "uds_path": "/tmp/cndp.sock",
```

Comment:
need to fix this up.

Reply:
This is not a straight forward change for lport-groups. updated known issues for now to unblock the PR